build.gradle

```diff
@@ -0,0 +1,27 @@
+buildscript {
```

Comment:
Can you add copyright in all  source files in this form:

// Copyright 2017 Sourcerer Inc. All Rights Reserved.